pydata_sphinx_theme/webpack-macros.html

```diff
@@ -0,0 +1,18 @@
+<!-- these macros are generated by "yarn build". do not edit by hand. -->
+{% macro head_pre() %}
+  <link rel="stylesheet"
+    href="{{ pathto('_static/vendor/fontawesome/5.13.0/css/all.min.css', 1) }}">
+  <link rel="stylesheet"
+    href="{{ pathto('_static/vendor/open-sans_all/1.44.1/index.css', 1) }}">
+  <link rel="stylesheet"
+    href="{{ pathto('_static/vendor/lato_latin-ext/1.44.1/index.css', 1) }}">
+  <link rel="stylesheet" href="{{ pathto('_static/css/index.17205a057956deafa7ca693bf7135e69.css', 1) }}">
+{% endmacro %}
+
+{% macro head_post() %}
+  <link rel="preload" as="script" href="{{ pathto('_static/js/index.0296141ff7d550c82fce.js', 1) }}">
+{% endmacro %}
+
+{% macro body_post() %}
+  <script src="{{ pathto('_static/js/index.0296141ff7d550c82fce.js', 1) }}"></script>
```

Comment:
My understanding is:
- in `head`, it starts preloading
- in `body`, it blocks until the loading finishes

In `master`, the JS isn't even considered to be needed until the whole page finishes loading (because javascript). this just allows the (increasingly large) index to get a head start.